.bashrc

```diff
@@ -80,8 +82,38 @@ if [ -x /usr/bin/dircolors ]; then
     alias egrep='egrep --color=auto'
 fi
 
+# Initiate virtualenv
 WORKON_HOME="/home/vagrant/.virtualenvs"
 source /usr/local/bin/virtualenvwrapper.sh > /dev/null 2>&1
 cd /vagrant
 
-# the workon command to be added during provisioning 
+# Ask for git configuration data from user based on input
+chmod 755 /home/vagrant/gitconfig
+export PATH=$PATH:/home/vagrant/
+echo 'export PATH=$PATH:/home/vagrant/' >> ~/.profile
+iterator1=0
+iterator2=0
+while [[ $iterator1 = 0 && $iterator2 -lt 5 && ! -f ~/.gitconfig ]]
+do
+    read -p 'Do you wish to setup Github credentials for easy access? (y/n) >>> ' github
+
+    if [ $github = 'y' ]
```

Comment:
Is there a standard snippet for that? I am not familiar with this type of tests but will have a look. If you have some resources for me to look at that would cut a lot of my research labour.